app/models/concerns/tags_substitution_concern.rb

```diff
@@ -238,6 +238,14 @@ def self.normalize(str)
     }
   ]
 
+  CONTACT_INFORMATION_NAME_TAG = {
+    id: 'dossier_contact_information_name',
+    libelle: 'nom du service instructeur',
+    description: 'Le nom du service qui traite le dossier (celui des informations de contact du groupe instructeur s’il existe, sinon celui de la démarche)',
+    lambda: -> (d) { d.service.nom || '' },
```

Comment:
```suggestion
    lambda: -> (d) { d.service&.nom || '' },
```

une démarche peut ne pas encore avoir de service / contact information (si elle n'est pas encore publiée par exemple)
Même si pour le moment tu as conditionné l'affichage à la présence d'un GI, ce n'est pas forcément celui du dossier et ça me paraît plus cohérent avec les autres tags